tests/test_models.py

```diff
@@ -89,3 +94,110 @@ def test_string_conversion(self, method):
             user_id=user_id
         )
         self.assertEqual(method(customer_user), expected_to_str)
+
+
+@mark.django_db
+@ddt.ddt
+class TestEnterpriseCustomerBrandingConfiguration(unittest.TestCase):
+    """
+    Tests of the EnterpriseCustomerBrandingConfiguration model.
+    """
+
+    def get_enterprise_customer(self):
+        """
+        @:return EnterpriseCustomer object.
+        """
+        faker = FakerFactory.create()
+        customer_uuid = faker.uuid4()
+        return EnterpriseCustomerFactory(uuid=customer_uuid, name="QWERTY")
```

Comment:
@asadiqbal08 seems unnecessary, uuid will be created in exact same way if not passed, and I don't think "QWERTY" is used anywhere else.